terraform/environments/electronic-monitoring-data/modules/lambdas/variables.tf

```diff
@@ -120,3 +120,9 @@ variable "function_tag" {
   nullable    = true
   default     = null
 }
+
+variable "ephemeral_storage_size" {
+  description = "Size in MB of lambda ephemeral storage"
+  type        = number
+  default     = 512
```

Comment:
This is the minimum, and I like the approach of ephemeral storage in principle... will this size be enough for all files encountered if they are sending us full file dumps?